x-pack/test/functional/apps/infra/hosts_view.ts

```diff
@@ -284,6 +294,16 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           });
         });
 
+        it('preserves selected tab between page reloads', async () => {
```

Comment:
It might be better to centralize these "preserves..." test scenarios in the `node_details` test suite. The `host_view` suite would become less focused on the asset details functionality, while the `node_details` could focus on more in-depth test cases. Wdyt?

Reply:
agree 👍